.github/workflows/build.yml

```diff
@@ -76,7 +76,7 @@ jobs:
             -DCMAKE_CXX_STANDARD=17 \
             -DCMAKE_BUILD_TYPE=RelWithDebInfo \
             -DCMAKE_INSTALL_PREFIX=/usr \
-            -DPYTHON_EXECUTABLE=$(command -v python3) \
+            -DPython_EXECUTABLE=$(command -v python3) \
```

Comment:
I think these could actually be removed, as CMake will find `/usr/bin/python3` just fine, but I can do this later.